components/charts/__tests__/ZoomTimeLine-test.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * Sonar UI Common
+ * Copyright (C) 2019-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+import { shallow } from 'enzyme';
+import * as React from 'react';
+// eslint-disable-next-line jest/no-mocks-import
+import { mockedTheme } from '../../__mocks__/mockedTheme';
```

Comment:
Would it be possible to disallow the `jest/no-mocks-import` rules globally for test files ?

Reply:
I think this rule apply only to test files, it's from jest plugin. I think the issue here is more that this theme file should maybe not be in a mock folder, because it doesn't mock anything that exists in SUC. So it's not a mock, it's just a default theme used for tests, but I don't know where we should put it...

Reply:
Ok don't worry about it ! Thanks.